web3/eth.py

```diff
@@ -85,68 +89,88 @@ def coinbase(self):
     def mining(self):
         return self.web3.manager.request_blocking("eth_mining", [])
 
+    @property
+    def hash_rate(self):
+        return self.web3.manager.request_blocking("eth_hash_rate", [])
+
     @property
     def hashrate(self):
-        return self.web3.manager.request_blocking("eth_hashrate", [])
+        raise DeprecationWarning("This method has been deprecated")
+
+    @property
+    def gas_price(self):
+        return self.web3.manager.request_blocking("eth_gas_price", [])
 
     @property
     def gasPrice(self):
-        return self.web3.manager.request_blocking("eth_gasPrice", [])
+        raise DeprecationWarning("This method has been deprecated")
 
     @property
     def accounts(self):
         return self.web3.manager.request_blocking("eth_accounts", [])
 
     @property
-    def blockNumber(self):
-        return self.web3.manager.request_blocking("eth_blockNumber", [])
+    def block_number(self):
+        return self.web3.manager.request_blocking("eth_block_number", [])
 
     @property
     def chainId(self):
         return self.web3.manager.request_blocking("eth_chainId", [])
 
-    def getBalance(self, account, block_identifier=None):
+    def get_balance(self, account, block_identifier=None):
         if block_identifier is None:
             block_identifier = self.defaultBlock
         return self.web3.manager.request_blocking(
-            "eth_getBalance",
+            "eth_get_balance",
```

Comment:
I believe this will still be `eth_getBalance`. I think it should be the actual JSON-RPC method name.